diff --git a/node_modules/react-native-svg/apple/RNSVGUIKit.h b/node_modules/react-native-svg/apple/RNSVGUIKit.h
index d8d4124..f4c8ba2 100644
--- a/node_modules/react-native-svg/apple/RNSVGUIKit.h
+++ b/node_modules/react-native-svg/apple/RNSVGUIKit.h
@@ -16,7 +16,7 @@
 #import <React/RCTUIManagerUtils.h>
 
 #ifdef RCT_NEW_ARCH_ENABLED
-#define RNSVGView RCTViewComponentView
+#define RNSVGView UIView
 #else
 #define RNSVGView UIView
 #endif 